docs/base/soa.md

```diff
@@ -0,0 +1,87 @@
+# Configuracions sincronització SOA
+
+Documentació general del mòdul de sincronització mitjançant fitxers SOA.
+
+## Menús
+
+El mòdul de sincronització proveeix dos llistats ubicats a "**_OpenERP Menu >
+Sync_**":
+
+- [Fitxers Importats](#fitxers-importats)
+- [Configuració](#configuracio)
+
+![](_static/soa/oerp_menu_soa.png)
+
+### Fitxers Importats
+
+En aquest menú es mostra un llistat dels fitxers de sincronització (SOA) importats.
+Des del llistat podem veure:
+
+- Data de la importació
+- Nom del fitxer importat
+- Tipus de sincrontizació:
+  - Sincronització massiva: Importació de moltes actualitzacions
```

Comment:
Add more spaces to show properly:

- Tipus de sincrontizació:
    - Sincronització massiva: Importació de moltes actualitzacions
    - Sincronització immediata: Importació d'una sola actualització

```
- Tipus de sincrontizació:
    - Sincronització massiva: Importació de moltes actualitzacions
    - Sincronització immediata: Importació d'una sola actualització
```